ts/receiver/dataMessage.ts

```diff
@@ -39,19 +41,21 @@ function cleanAttachment(attachment: any) {
   };
 }
 
-function cleanAttachments(decrypted: SignalService.DataMessage) {
-  const { quote } = decrypted;
+function cleanAttachments(decryptedDataMessage: SignalService.DataMessage) {
+  const { quote } = decryptedDataMessage;
 
   // Here we go from binary to string/base64 in all AttachmentPointer digest/key fields
 
-  // we do not care about group on Session
+  // we do not care about group on Session Desktop
```

Comment:
Still true?

Reply:
yes, changed to
  // we do not care about the group field on Session Desktop